src/main/java/com/dreamsportslabs/guardian/utils/Utils.java

```diff
@@ -37,4 +42,25 @@ public static MultiMap getForwardingHeaders(MultivaluedMap<String, String> heade
   public static String getMd5Hash(String input) {
     return DigestUtils.md5Hex(input).toUpperCase();
   }
+
+  public static String generateState() {
```

Comment:
These two util functions might make more sense in OtpUtils rather than generic Utils.